### PR TITLE
Update blogs.yml, Added Jerry2001 to blog section

### PR DESCRIPTION
**Pull request by NgoMinhChau**
My task: https://codein.withgoogle.com/dashboard/task-instances/4972888325095424/
My gh-page https://github.com/NgoMinhChau/gci16.fossasia.org/tree/gh-pages/_data

### CODE REVIEW

**Comment by m1guelpf** [comment]
@NgoMinhChau Please share your live link.

File: _data/blogs.yml [hunk 1]
```diff
@@ -51,3 +51,8 @@
   blog_img:
   article: GCI 2016
   article_link: http://dcongtinh.blogspot.com/2016/12/google-code-in-2016-google-code-in.html
+
```
Comment (m1guelpf) [inline _data/blogs.yml]: Remove trailing space in line 54

**Comment by m1guelpf** [comment]
LGTM

**Comment by NgoMinhChau** [comment]
thank you so much

**Comment by m1guelpf** [comment]
@NgoMinhChau No problem :smile:

**Pull request merged**